mmv1/third_party/terraform/website/docs/d/cloudfunctions2_function.html.markdown

```diff
@@ -0,0 +1,187 @@
+---
+subcategory: "Cloud Functions (2nd gen)"
+page_title: "Google: google_cloudfunctions2_function"
+description: |-
+  Get information about a Google Cloud Function (2nd gen).
+---
+
+# google\_cloudfunctions2\_function
+
+Get information about a Google Cloud Function (2nd gen). For more information see:
+
+* [API documentation](https://cloud.google.com/functions/docs/reference/rest/v2beta/projects.locations.functions).
+
+## Example Usage
+
+```hcl
+data "google_cloudfunctions2_function" "my-function" {
+  name = "function"
+  location = "us-central1"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of a Cloud Function (2nd gen).
+
+* `location` - (Required) The location in which the resource belongs.
+
+- - -
+
+* `project` - (Optional) The project in which the resource belongs. If it
+    is not provided, the provider project is used.
+
+## Attributes Reference
```

Comment:
Can we link out to the resource page, similar to https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/sql_database_instance#attributes-reference? We're not realistically going to keep this up-to-date, and maintaining a listing of fields will imply new fields are not supported.

Reply:
Sure, I can do that.